src/main/java/co/kr/muldum/adapter/in/web/dto/ErrorResponse.java

```diff
@@ -0,0 +1,24 @@
+package co.kr.muldum.adapter.in.web.dto;
+
+public class ErrorResponse {
+
+    private final String errorCode;
+    private final String message;
+
+    private ErrorResponse(String errorCode, String message) {
+        this.errorCode = errorCode;
+        this.message = message;
+    }
+
+    public static ErrorResponse of(String errorCode, String message) {
+        return new ErrorResponse(errorCode, message);
+    }
```

Comment:
여기서 이걸 두개로 나눠서 오버로딩을 한 이유를 모르겠어요 오류코드와 오류를 반환할때
3개의 요소를 모두 포함한(error, statusCode, message) 하나의 메서드면 충분할것 같아요

혹은 stuatusCode와 message 두개만 반환하던가 하는게 좋을것 같아요.
아니면 명확하게
```suggestion
    public static ErrorResponse ofError(String error, String message) {
        ErrorResponse response = new ErrorResponse();
        response.error = error;
        response.message = message;
        return response;
    }

    public static ErrorResponse ofStatus(Integer statusCode, String message) {
        ErrorResponse response = new ErrorResponse();
        response.statusCode = statusCode;
        response.message = message;
        return response;
    }
```
그리고
```java
ErrorResponse response = new ErrorResponse();
        response.error = error;
        response.message = message;
        return response;
```
보단
```java
return new ErrorResponse(error, null, message)
```
이런식이 더 좋을것 같아요

Reply:
이미 수정된 부분이여서 다시 확인해보시면 될 거 같아요